### PR TITLE
BaSP-TG-59: create, delete and edit employees and projects from admins

### PR DESCRIPTION
**Pull request by Maxi-SR**
- Use Projects and Employees from global Store (using Redux) in index entity.
- Use Projects and Employees from global Store (using Redux) in Form entity.
- Add routes in Layout/index to make projects and employees CRUD's from admins.

How to test:
- In admins page you can add, delete and update a project or an employee depending on what you choose from buttons (employees or projects)

### CODE REVIEW

File: src/Components/Admins/index.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Table from '../Shared/Table';
+import styles from './admins.module.css';
+import Button from '../Shared/Button';
+import { getEmployees } from '../../redux/employees/thunks';
+import { getProjects } from '../../redux/projects/thunks';
```
Comment (DamianPalavecino) [inline src/Components/Admins/index.jsx:7]:
```suggestion
import { getEmployees, deleteEmployee } from '../../redux/employees/thunks';
import { getProjects, deleteProject } from '../../redux/projects/thunks';
```
And delete lines 11 and 12

File: src/Components/Admins/index.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        ...showModal,
+        [modal]: !showModal[modal]
+      });
+    }
+  };
+
+  const editEntity = (id) => {
+    entity === 'employees'
+      ? history.push(`employees/form/${id}`)
+      : history.push(`projects/form/${id}`);
```
Comment (DamianPalavecino) [inline src/Components/Admins/index.jsx:55]:
```suggestion
    history.push(`${entity}/form/${id}`);
```

File: src/Components/Admins/index.jsx (hunk 1 above)
Comment (Maxi-SR) [reply src/Components/Admins/index.jsx:7]: thanks

File: src/Components/Admins/index.jsx (hunk 2 above)
Comment (Maxi-SR) [reply src/Components/Admins/index.jsx:55]: okey, thanks

**Review by Juanoauil — APPROVED** [review]

**Review by DamianPalavecino — APPROVED** [review]
Approved, well done

**Comment by SteampunkMatu** [comment]
Looks good for me

**Pull request merged**